automated-testing/performance-testing/readme.md

```diff
@@ -1,67 +1,286 @@
+| Paging File(_Total)\% Usage | The percentage of the system page file that is currently in use. This is not directly related to performance, but you can run into serious application issues if the page file does become completely full and additional memory is still being requested by applications.  |
 
 ## Key Performance testing activities
 
-  Performance testing activities vary depending on sub category of performance test, the engagement requirements and constraints. For specific guidance you can follow the link to the sub category of performance tests listed above. Following are some activities which will generally be involved:
+Performance testing activities vary depending on sub category of performance
+test, the engagement requirements and constraints. For specific guidance you can
+follow the link to the sub category of performance tests listed above. Following
+are some activities which will generally be involved:
 
 ### Identify and Define the Acceptance criteria for the tests
 
-  This will generally include identifying and defining the goals and constrains for the performance characteristics of the system
-  
+This will generally include identifying and defining the goals and constrains
+for the performance characteristics of the system
+
 ### Plan and design the tests
-  
-  In general we need to consider the following points:
-
-* Defining the load the application would be tested with
-* Establishing the metrics to be collected
-* Establish which tools will be used for the tests
-* Establish the performance test frequency : whether the performance tests be done as a part of the feature development sprints, or only prior to release to a major environment?
-  
+
+In general we need to consider the following points:
+
+- Defining the load the application would be tested with
+
+- Establishing the metrics to be collected
+
+- Establish which tools will be used for the tests
+
+- Establish the performance test frequency : whether the performance tests be
+  done as a part of the feature development sprints, or only prior to release to
+  a major environment?
+
 ### Test implementation
 
+- Apply the design and create the performance tests based on them
+
 ### Test Execution
 
+- Execute the tests and collect the performance metrics. Take note of the
+  previous section Around Load Testing, Chaos Testing, Stress and Spike Testing
+
 ### Result analysis and re-testing
 
-* The test are executed, the results are collected and the environments are monitored
-* The results are analysed
-* Depending on the scenario, modification of application or configuration are done and testing cycle is repeated again.
+- The test are executed, the results are collected and the environments are monitored
+
+- The results are analysed
+
+- Depending on the scenario, modification of application or configuration are
```

Comment:
Follow the same pattern as in the earlier sections with active sentences. i.e. Depending on the scenario, modify the application or configuration and repeat the tests.